src/igvfd/searches/configs/construct_library_set.py

```diff
@@ -22,6 +22,36 @@ def construct_library_set():
             'selection_criteria': {
                 'title': 'Selection Criteria'
             },
+            'small_scale_gene_list.symbol': {
+                'title': 'Targeted Genes'
```

Comment:
I think in terms of code this is fine, but it is throwing me off slightly that this facet is called Targeted Genes, meanwhile in Measurement Set we have the property "targeted_genes" with no relation to this. But I'm not sure I have a better replacement facet title either.

Reply:
I see. I think it is ok to leave it like this, just threw me off a bit. But we can always reevaluate later if there are more people getting confused.

Reply:
I think we should firmly distinguish the difference between assessed_genes, targeted_genes, and small_scale_gene_list/large_scale_gene_list. But I think this can accomplished in a follow up ticket which will also update the description. Between its usage on prediction set and CLS it seems small_scale_gene_list/large_scale_gene_list is more of an "input" and the former two are "outputs". Hope that is a start.